.evergreen/run-tests.sh

```diff
@@ -28,12 +26,23 @@ if [ "$SSL" = "yes" ]; then
    fi
 fi
 
+if [ "${SKIP_CRYPT_SHARED}" = "yes" ]; then
+   CRYPT_SHARED_LIB_PATH=""
+   echo "crypt_shared library is skipped"
+elif [ -z "${CRYPT_SHARED_LIB_PATH}" ]; then
+   echo "crypt_shared library path is empty"
+else
+   echo "crypt_shared library will be loaded from path: $CRYPT_SHARED_LIB_PATH"
+fi
+
 echo "Running tests with URI: $MONGODB_URI"
 
 # Run the tests, and store the results in a junit result file
-case "$OS" in
-   *)
-      API_VERSION="${API_VERSION}" TEST_PHP_JUNIT="${PROJECT_DIRECTORY}/test-results.xml" TEST_PHP_ARGS="-q -x --show-diff -g FAIL,XFAIL,BORK,WARN,LEAK,SKIP" make test TESTS=$TESTS
-      ;;
-esac
-
+API_VERSION="${API_VERSION}" \
+CRYPT_SHARED_LIB_PATH="${CRYPT_SHARED_LIB_PATH}" \
+MONGODB_URI="${MONGODB_URI}" \
+SSL_DIR="${SSL_DIR}" \
+TEST_PHP_JUNIT="${PROJECT_DIRECTORY}/test-results.xml" \
+TEST_PHP_ARGS="-q -x --show-diff -g FAIL,XFAIL,BORK,WARN,LEAK,SKIP" \
+TESTS="$TESTS" \
+make test
```

Comment:
Although `make` does allow env vars to be specified _after_ the command, I opted to put everything up front so we have everything on its own line as was done in the Evergreen task definition.